docs/model_serving_framework/text_embedding_model_examples.md

```diff
@@ -93,6 +111,39 @@ GET /_plugins/_ml/tasks/zgla5YUB1qmVrJFlwzW-
     "is_async": true
 }
 ```
+
+We can also register model from URL. To do that we need to update the following cluster settings:
+
+```
+PUT _cluster/settings
```

Comment:
Reformat this part by adding indent, also the response

Reply:
addressed.